lib/karaokium_web/live/admin/performance_live/show.ex

```diff
@@ -15,9 +16,65 @@ defmodule KaraokiumWeb.Admin.PerformanceLive.Show do
      socket
      |> assign(:page_title, page_title(socket.assigns.live_action))
      |> assign(:karaoke_id, karaoke_id)
-     |> assign(:performance, Performances.get_performance!(id))}
```

Comment:
Whyt are you deleting this one and creating a `:performancelive` that is an id?

Reply:
If you look down on this function, it will call the function reload that will do the same thing. I did this to make the code more eligible. It is only 1 line, but I tried to make the same thing as on show.ex on the karaoke_live/ folder .